js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -450,16 +450,18 @@ exports.ColumnsView = modules.View.inherit(columnStateMixin).inherit({
     _renderCell: function($row, options) {
         var that = this,
             cellOptions = that._getCellOptions(options),
-            column = options.column,
             $cell;
 
         options.row.cells.push(cellOptions);
 
         $cell = that._createCell(cellOptions);
 
-        // TODO move to cellPrepared
-        if(!column.command) {
-            that.setAria("label", that.localize("dxDataGrid-ariaColumn") + " " + column.caption + ", " + that.localize("dxDataGrid-ariaValue") + " " + cellOptions.text, $cell);
+        if(cellOptions.rowType !== "freeSpace") {
+            that.component.setAria({
```

Comment:
Do not use that.component.setArea. The that.setAria is faster.

Reply:
Ok, than I need to invoke 3 times "that.setAria"